packages/api/lib/pdrHelpers.js

```diff
@@ -17,6 +18,61 @@ function generateShortPAN(disposition) {
   );
 }
 
+/**
+ * Get list of input granules from execution
+ *
+ * @param {string} executionArn - execution arn
+ * @returns {object[]} list of granules
+ */
+async function getGranulesFromExecution(executionArn) {
+  const excObj = await getExecution({
+    prefix: process.env.stackName,
+    arn: executionArn,
+  });
+  return excObj.originalPayload.granules;
```

Comment:
When typing is updated, I'd assume this should use optional chaining *as we can't be certain the original payload has a granules object* in the context of this method as an exported thing.  

That context concern makes me further think this should be denoted as an internal method and/or update the docstring